plugins/node/opentelemetry-instrumentation-aws-lambda/package.json

```diff
@@ -63,10 +63,10 @@
     "typescript": "3.9.7"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.18.1",
-    "@opentelemetry/instrumentation": "^0.18.2",
-    "@opentelemetry/resources": "^0.18.2",
-    "@opentelemetry/semantic-conventions": "^0.18.2",
-    "@opentelemetry/tracing": "^0.18.2"
+    "@opentelemetry/api": "^1.0.0-rc.0",
```

Comment:
I think we should move API to peer dependencies first to stay consistent with core.

Reply:
I am trying to debug an issue here. that's why its a draft. my local npm install is failing.

Reply:
npm 6 or 7?

Reply:
6.14.11

```
lerna ERR! npm install exited 1 in '@opentelemetry/instrumentation-redis'
lerna ERR! npm install stderr:
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @opentelemetry/semantic-conventions@^0.19.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'opentelemetry-instrumentation-redis'
npm ERR! notarget 
```

Happens on all packages that depend on semantic conventions